pkgs/by-name/ko/komac/package.nix

```
@@ -53,6 +57,13 @@ rustPlatform.buildRustPackage {
   nativeInstallCheckInputs = [ versionCheckHook ];
   versionCheckProgram = "${placeholder "out"}/bin/komac";
 
+  postInstall = ''
```

To enable cross compilation, we have to skip generating these in some cases. 

```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

---

Thanks for pointed out 🙏 Updated in https://github.com/NixOS/nixpkgs/pull/362157/commits/a8fd645b081993fc61264d7ba48cad748cfa39df
I think, I should learn this thing in https://github.com/NixOS/nixpkgs/pull/356548, but I didn't actually understand 🙇‍♂️ https://github.com/NixOS/nixpkgs/blob/e19e156796ad04a533f9bbaba27f13289783c137/doc/hooks/installShellFiles.section.md?plain=1#L87-L110